app.js

```diff
@@ -20,7 +26,9 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cleanStrings);
-app.use('/api/v1', router);
+
+app.use('/api/articles', articleRouter);
```

Comment:
This isn't very clean to me.  We should only be exporting one instance of the router into this file. We should have an index router file that encapsulates all the endpoints for our entities. Something like this. 
```
export default (router) => {
    article(router);
    user(router)
};
```
Where article and user are instances of the article and user routes. Let me know if you have questions

Reply:
Noted